apps/ejabberd/src/mod_mam_iq.erl

```diff
@@ -0,0 +1,140 @@
+-module(mod_mam_iq).
```

Comment:
I like the idea. I can rename `mod_mam_iq` to `mam_iq`. Regarding `mod_mam_utils` - we should rename it as well. In other PR though, I may simply don't have enough time for that...